src/common/guided_filter.c

```diff
@@ -55,6 +55,18 @@
 # define _mm_prefetch(where,hint)
 #endif
 
+#ifndef dt_omp_shared
+#ifdef _OPENMP
+#if defined(__clang__) || __GNUC__ > 8
+# define dt_omp_shared(var, ...)  shared(var, __VA_ARGS__)
```

Comment:
I suppose this should be:
`# define dt_omp_shared(...)  shared(__VA_ARGS__)`

To allow for a single argument to be passed.

Reply:
You're right, though interestingly enough GCC9 is fine with that definition and a single arg.  I had just copied the definition I used in common/nlmeans_core.c; now that it's used in two places, it should probably be refactored into a header file (the obvious place being right after the definition of dt_omp_firstprivate in common/darktable.h).

Reply:
I'll try to do that indeed.